lib/python/ensembl/brc4/runnable/integrity.py

```diff
@@ -297,17 +301,19 @@ def parse_gff3(self, gff3_handle):
         genes = {}
         peps = {}
         all_peps = {}
+        tes = {}
 
         gff = GFF.parse(gff3_handle)
         for seq in gff:
             seqs[seq.id] = len(seq.seq)
             
             for feat in seq.features:
+                feat_length = abs(feat.location.end - feat.location.start)
```

Comment:
"Important" question: are the start and end supposed to be both part of the feature? If so, a `+1` would be required to get the exact length (I have always despised these random decisions on whether `end` is actually the last character or the next one :unamused:)

Reply:
This length is compared with the one calculated from the peptide fasta length, so the comparison should have raised errors quickly if we didn't... assuming the code that compares the lengths does so correctly.
I will need to review that part of the code to make sure (a good example of why we need some tests); though that's not the purpose of this PR.